kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -712,6 +721,8 @@ var ExercisePracticeView = ExerciseWrapperBaseView.extend({
         });
 
         this.listenTo(this.exercise_view, "check_answer", this.check_answer);
+
+        this.listenTo(this.exercise_view, "hint_available", this.toggle_hint_view);
```

Comment:
`toggle_hint_view` takes an argument, but my understanding is that this form will not call toggle_hint_view with an argument. Is that correct? Can you use [_.partial](http://underscorejs.org/#partial) here?